pkg/chartutil/load.go

```diff
@@ -118,7 +118,7 @@ func loadArchiveFiles(in io.Reader) ([]*BufferedFile, error) {
 		n = path.Clean(n)
 		if n == "." {
 			// In this case, the original path was relative when it should have been absolute.
-			return nil, errors.New("chart illegally contains empty path")
+			return nil, fmt.Errorf("chart illegally contains content outside the base directory: %q", hd.Name)
```

Comment:
I think it might read better as follows:
""chart illegally contains content outside the base directory" --> ""Chart content is outside of the base directory"

Reply:
@hickeyma I actually think this is ok for now. It matches previous language anyway, but up to you if you'd like to see the change

Reply:
Yeah, especially since there are multiple errors similarly starting with "chart illegally..." I think this is fine as well. We can change this holistically in Helm 3 if need be though :)